pkg/persistentvolumeclaim/debug_cstor.go

```diff
@@ -183,3 +181,25 @@ func resourceStatus(crs util.CstorVolumeResources) error {
 
 	return nil
 }
+
+func displayPVCEvents(k client.K8sClient, crs util.CstorVolumeResources) error {
+	// 1. Set the namespace of the resource to the client
+	k.Ns = crs.PVC.Namespace
+	// 2. Fetch the events of the concerned PVC
+	events, err := k.GetEvents(fmt.Sprintf("regarding.name=%s,regarding.kind=PersistentVolumeClaim", crs.PVC.Name))
+	// 3. Display the events
+	if len(events.Items) != 0 && err == nil {
+		_, _ = fmt.Fprint(os.Stdout, "\nChecking PVC Events:", util.ColorText(fmt.Sprintf(" %s %d! ", util.UnicodeCross, len(events.Items)), util.Red), "-------->\n")
+		var crStatusRows []metav1.TableRow
+		for _, event := range events.Items {
+			crStatusRows = append(crStatusRows, metav1.TableRow{Cells: []interface{}{event.Action, event.Reason, event.Note, util.ColorStringOnStatus(event.Type)}})
+		}
+		defer util.TablePrinter(util.EventsColumnDefinitions, crStatusRows, printers.PrintOptions{})
```

Comment:
Can defer be dropped, as this is the last condition anyway & it's not like a cleanup task,

Ref: [Blog on defer](https://blog.golang.org/defer-panic-and-recover)

Reply:
Yes, this needs to be removed.